docs/release-notes/8.18.asciidoc

```diff
@@ -0,0 +1,142 @@
+[[release-notes-header-8.18.0]]
+== 8.18
+
+[discrete]
+[[release-notes-8.18.0]]
+=== 8.18.0
+
+[discrete]
+[[known-issue-8.18.0]]
```

Comment:
Apologies as this is last minute, but created a ticket about needing to doc [this](https://github.com/elastic/docs-content/issues/1021) as a known issue. Would it be possible to add here?

Reply:
Added in [9f38601](https://github.com/elastic/security-docs/pull/6519/commits/9f38601fdcaf44f9df30d82a31061fd06ea8c608).